TypeCobol.Test/Parser/Programs/Cobol85/ParagraphSection.rdzPGM.txt

```diff
@@ -1,10 +1,37 @@
 --- Diagnostics ---
-Line 15[8,28] <37, Warning, General> - Warning: Section 'SectionEmpty' is empty
-Line 17[8,29] <37, Warning, General> - Warning: Section 'SectionEmpty2' is empty
-Line 26[8,14] <37, Warning, General> - Warning: Paragraph 'Parag2' is empty
-Line 34[8,14] <37, Warning, General> - Warning: Paragraph 'Parag4' is empty
-Line 37[8,14] <37, Warning, General> - Warning: Paragraph 'Parag5' is empty
-Line 41[8,14] <37, Warning, General> - Warning: Paragraph 'Parag6' is empty
+Line 19[20,24] <30, Error, Semantics> - Semantic error: Ambiguous reference to paragraph PAR05 OffendingSymbol=[20,24:PAR05]<UserDefinedWord>
+Line 28[20,24] <30, Error, Semantics> - Semantic error: Ambiguous reference to paragraph PAR04 OffendingSymbol=[20,24:PAR04]<UserDefinedWord>
+Line 31[20,24] <30, Error, Semantics> - Semantic error: Ambiguous reference to procedure SEC05 OffendingSymbol=[20,24:SEC05]<UserDefinedWord>
+Line 37[20,24] <30, Error, Semantics> - Semantic error: Ambiguous reference to section SEC07 OffendingSymbol=[20,24:SEC07]<UserDefinedWord>
+Line 45[20,24] <30, Error, Semantics> - Semantic error: Ambiguous reference to paragraph PAR05 OffendingSymbol=[20,24:PAR05]<UserDefinedWord>
+Line 54[20,24] <30, Error, Semantics> - Semantic error: Ambiguous reference to paragraph PAR04 OffendingSymbol=[20,24:PAR04]<UserDefinedWord>
+Line 66[8,13] <27, Error, Syntax> - Syntax error : Paragraph 'PAR05' already declared in procedure-division
```

Comment:
Cobol allow you to define 2 paragraphs with the same name.
So this should not be an error but rather a warning.

Reply:
Changed to a Warning.